applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/shell_thin_adjoint_element_3D3N.cpp

```diff
@@ -6,49 +6,39 @@
 //  License:		 BSD License
 //					 license: structural_mechanics_application/license.txt
 //
-//  Main authors:    Martin Fusseder
+//  Main authors:    Martin Fusseder, https://github.com/MFusseder 
 //
 
 #include "shell_thin_adjoint_element_3D3N.hpp"
 #include "custom_utilities/shellt3_corotational_coordinate_transformation.hpp"
 #include "structural_mechanics_application_variables.h"
+#include "custom_response_functions/response_utilities/response_data.h"
+#include "includes/checks.h"
 
 //#include "geometries/triangle_3d_3.h"
 
 #include <string>
 #include <iomanip>
 
-#define OPT_NUM_NODES 3 //------------------------->TODO: check which of the stuff is here really needed
-#define OPT_STRAIN_SIZE 6
-#define OPT_NUM_DOFS 18
-
 //----------------------------------------
 // preprocessors for the integration
 // method used by this element.
 
 //#define OPT_1_POINT_INTEGRATION
 
-#ifdef OPT_1_POINT_INTEGRATION
-#define OPT_INTEGRATION_METHOD Kratos::GeometryData::GI_GAUSS_1
-#define OPT_NUM_GP 1
-#else
-#define OPT_INTEGRATION_METHOD Kratos::GeometryData::GI_GAUSS_2
-#define OPT_NUM_GP 3
-#endif // OPT_1_POINT_INTEGRATION
-
 //----------------------------------------
 // preprocessors to handle the output
 // in case of 3 integration points
 
-#define OPT_USES_INTERIOR_GAUSS_POINTS
+//#define OPT_USES_INTERIOR_GAUSS_POINTS
```

Comment:
wsn't this removed or being removed by @philbucher ?

Reply:
I am quite sure I did but I have to check

Since there was a lot of work to be done for this PR I haven't reviewed properly yet

@MFusseder @armingeiser when you are ready let me know then I will make a proper review